src/components/ButtonGroupPicker/__test__/buttonGroupPicker.spec.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { Component as ButtonGroupPicker } from '../index';
+import ButtonOption from '../../ButtonOption/index';
+
+describe('<ButtonGroupPicker />', () => {
+    it('should render a label when label prop is passed', () => {
+        const component = mount(<ButtonGroupPicker label="My label" />);
+        expect(component.find('legend').length).toBe(1);
+    });
+
+    it('should render bottom help text when bottomHelpText prop is passed', () => {
+        const component = mount(<ButtonGroupPicker bottomHelpText="My help text" />);
+        expect(component.find('div[children="My help text"]').length).toBe(1);
+    });
+
+    it('should render required asterisk when required prop is true', () => {
+        const component = mount(<ButtonGroupPicker label="My label" required />);
+        expect(component.find('abbr[title="required"]').length).toBe(1);
```

Comment:
since abbr is an implementation detail of the RequiredAsterisk component we should treat it as a black box, the we should do:
```expect(component.find('RequiredAsterisk').exists()).toBe(true);```